deploy/helm/preview.yml

```diff
@@ -5,7 +5,7 @@ global:
     imagePullSecret: "regcred"
     pullPolicy: Always
   exposeNodePorts: false
-  enablePriorityClass: true
+  enablePriorityClass: false
```

Comment:
I had problems with this in its current form - most of the deployments wouldn't even create pods because they couldn't find the priority classes. I think having no priority classes for previews is fine as long as we keep them enabled for dev?

Reply:
Yeah~ I think it's fine to have `enablePriorityClass` to false.
The missing `enablePriorityClass` I think it's removed by other PR / Preview.
Ideally, if we really want both preview namespaces & dev have this enabled, we probably only want to the `priorityClass` to be installed by dev and all preview namespaces just reference the created `priorityClass`. In this way, we won't have the troubles of `priorityClass` being removed  when a preview site is stopped.
Having said that, I think disable it definitely make it safer~